package.json

```diff
@@ -12,6 +12,9 @@
     "prestart": "webpack",
     "start": "webpack-dev-server"
   },
+  "bin": {
+    "enclave": "index.js"
+  },
```

Comment:
This package.json config tells npm to add an `enclave` link in `node_modules/.bin` that points to `index.js`.  Since `node_modules/.bin` is added to the `PATH` during `npm run x`, this allows your `index.js` script to be ran under the name `enclave`.